pymc3/distributions/multivariate.py

```diff
@@ -737,6 +738,25 @@ def __str__(self):
 matrix_pos_def = PosDefMatrix()
 
 
+class WishartRV(RandomVariable):
+    name = "wishart"
+    ndim_supp = 2
+    ndims_params = [0, 2]
+    dtype = "floatX"
+    _print_name = ("Wishart", "\\operatorname{Wishart}")
+
+    def _shape_from_params(self, dist_params, rep_param_idx=1, param_shapes=None):
+        # The shape of second parameter `V` defines the shape of the output.
+        return param_shapes[1]
```

Comment:
Does this make sense like this @brandonwillard, I am not very familiar with the signature.

Reply:
We can also use `dist_params[1].shape`. Just needs to be same as shape of second argument `V`.